### ISSUE TITLE
Cumulative elevation loss

### ISSUE THREAD

**Issue by carlos-mg89** [Sibyx/phpGPX#39]
Thanks for this wonderful library! It definitely eases the parsing of GPX to obtain it's related data.

I have noticed that you include the cumulative elevation gain, however I'm unable to see or to obtain the cumulative elevation loss, which is not as important for many people but it still is a decisive factor for some hikers who have legs' pain while descending tracks.

**Comment by carlos-mg89** [comment]
Actually, I wrote that down cause I didn't see that field in the data I obtained, only the gain. However, while checking the github repo I see this field exists here: https://github.com/Sibyx/phpGPX/blob/master/src/phpGPX/Models/Stats.php

The documentation doesn't talk about it: https://sibyx.github.io/phpGPX/phpGPX-Models-Stats.html

And if I try to read it in the 1.0-RC5, it just doesn't exist, although it's the latest release. Will try with the latest commit.
